src/lib/pythongen.ml

```diff
@@ -35,42 +45,55 @@ class Rpc_light_failure(Exception):
 
 
 class Unimplemented(Rpc_light_failure):
+    """The called RPC method is not implemented."""
     def __init__(self, arg):
         super(Unimplemented, self).__init__(self, "Unimplemented", [arg])
 
 
 class InternalError(Rpc_light_failure):
+    """
+    Indicates either an error in the generated bindings to dispatch the RPC
+    call, or an unknown exception raised by the implementation that is not
+    declared in the interface.
+    """
     def __init__(self, error):
         super(InternalError, self).__init__(self, "Internal_error", [error])
 
 
 class UnmarshalException(InternalError):
+    """The input does not have the expected structure."""
     def __init__(self, thing, ty, desc):
         super(UnmarshalException, self).__init__(
             self,
             "UnmarshalException thing=%s ty=%s desc=%s" % (thing, ty, desc))
 
 
 class TypeError(InternalError):
+    """The type of an argument differs from the one defined in the interface."""
     def __init__(self, expected, actual):
         super(TypeError, self).__init__(
             self, "TypeError expected=%s actual=%s" % (expected, actual))
 
 
 class UnknownMethod(InternalError):
+    """The called method is not defined in the interface."""
     def __init__(self, name):
         super(UnknownMethod, self).__init__(self, "Unknown method %s" % name)
 
 
 class ListAction(argparse.Action):
+    """
+    Custom argparse action for specifying dictionaries on the command line.
+    The syntax is <prog> --dict_name key1 value1 --dict_name key2 value2 ...
+    """
     def __call__(self, parser, namespace, values, option_string=None):
-        k = values[0]
-        v = values[1]
+        key = values[0]
+        value = values[1]
         if ((hasattr(namespace, self.dest) and
```

Comment:
You don't need the double parentheses here

Reply:
Good spot, I also had to indent it